tests/cypress/integration/issue_1919_check_text_attr.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Check label attribute changes', () => {
+
+    const issueId='1919'
+    const labelName=`Issue ${issueId}`
+    const taskName=`New annotation task for ${labelName}`
+    const attrName=`Attr for ${labelName}`
+    const textDefaultValue='Some default value for type Text'
+    const image=`image_${issueId}.png`
+    const newLabelAttrValue = 'New attribute value'
+    const width='800'
+    const height='800'
```

Comment:
Why we use strings instead of numbers here?

Reply:
Jimp successfully handles this format as well. But it is better to use a numeric format. Thanks.